src/test/java/net/masterthought/cucumber/generators/integrations/FeatureReportPageIntegrationTest.java

```diff
@@ -304,12 +304,15 @@ public void generatePage_generatesEmbedding() {
         Embedding[] embeddings = feature.getElements()[0].getSteps()[5].getEmbeddings();
 
         assertThat(embeddingsElement).hasSameSizeAs(embeddings);
+        embeddingsElement[0].getLinks()[0].hasLabelAndAddress(embeddings[0].getName(), "");
         embeddingsElement[0].hasImageContent(embeddings[0]);
-        asserEmbeddingFileExist(embeddings[0]);
+        assertEmbeddingFileExist(embeddings[0]);
+        embeddingsElement[2].getLinks()[0].hasLabelAndAddress("Attachment 3 (Plain text)", "");
         embeddingsElement[2].hasTextContent(embeddings[2].getData());
-        asserEmbeddingFileExist(embeddings[2]);
+        assertEmbeddingFileExist(embeddings[2]);
+        embeddingsElement[3].getLinks()[0].hasLabelAndAddress(embeddings[3].getName(), "");
```

Comment:
why address is empty?

Reply:
It's because `<a>` element for attachment name does not contain `href` attribute. In such  case method `hasLabelAndAddress` in `LinkAssertion` returns an empty String when evaluating `element.attr("href")`